Documentation/Global/Form/Checkbox.rst

```diff
@@ -9,46 +9,224 @@ Form.checkbox ViewHelper `<f:form.checkbox>`
 
 ..  typo3:viewhelper:: form.checkbox
     :source: ../../Global.json
-    :display: tags,description,gitHubLink,arguments
+    :display: tags,description,gitHubLink
+    :noindex:
+
+..  contents:: Table of contents
+
+..  _typo3-fluid-form-checkbox-single:
+
+Single checkboxes
+=================
+
+A single checkbox is usually mapped to a boolean property or controller
+argument.
+
+However due to how checkboxes work in HTML a non-checked checkbox does not
+appear in the request send by the form. Therefore properties and arguments
+need to default to `false` they will then be set to `true` if the checkbox
+was checked.
+
+While the :ref:`value <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-form-checkboxviewhelper-value>`
+does not play a role if you are working with a checkbox the argument must still
+be supplied.
 
 ..  _typo3-fluid-form-checkbox-example:
 
-Examples
-========
+Single optional checkbox tied to a action argument
+--------------------------------------------------
+
+Checkboxes in HTML work a little different from other input fields in that
+multiple or none at all can be checked.
+
+Fluid outputs a normal HTML `<input type="checkbox" <https://www.w3schools.com/tags/att_input_type_checkbox.asp>`_
+and some of the pitfalls also apply here. Especially a check box that is not
```

Comment:
Does that render properly with < at the start? Can the input tag be closed?

Reply:
well the tag itself would not be finishes but contain more. and yes, should render fine it is htmlspecialchared

Reply:
how about the suggestion above then, to indicate the whole tag?